front/src/applications/stdcm/utils/formatStdcmConfV2.ts

```diff
@@ -15,6 +15,7 @@ import { dateTimeToIso } from 'utils/date';
 import { ISO8601Duration2sec, sec2ms, time2sec } from 'utils/timeManipulation';
 
 import createMargin from './createMargin';
+import { mToMm } from 'utils/physics';
```

Comment:
It seems to me that we need to move imports to solve the prettier problem.
`
import { mToMm } from 'utils/physics';
import { ISO8601Duration2sec, sec2ms, time2sec } from 'utils/timeManipulation';
`

Reply:
You're right, thanks :+1:

Reply:
Done !